test/unit/model/changes-utils.test.ts

```diff
@@ -297,17 +297,98 @@ describe('cloneManuscript', () => {
+      "path": "relatedArticles",
+      "idField": "id",
+      "object": {
+        "_id": "ad319b14-c312-4627-a5a1-d07a548a6e7e",
+        "articleType": "article-reference",
+        "href": "111111"
+      }
+    }];
+
+    const updatedManuscript = applyChangesToManuscript(mockManuscript, changesJSON);
+    expect(updatedManuscript.relatedArticles.length).toBe(1);
+    expect(updatedManuscript.relatedArticles[0]).toEqual(changesJSON[0].object);
+  });
+
+  it('deletes an entity from related article list', () => {
+    const changesJSON = [{
+      "type": "add-object",
+      "timestamp": 161409771111,
+      "path": "relatedArticles",
+      "idField": "id",
+      "object": {
+        "_id": "ad319b14-c312-1111-a5a1-d07a548a6e7e",
+        "articleType": "article-commentary",
+        "href": "222"
+      }
+    }, {
+      "type": "add-object",
+      "timestamp": 1614097785693,
+      "path": "relatedArticles",
+      "idField": "id",
+      "object": {
+        "_id": "ad319b14-c312-4627-a5a1-d07a548a6e7e",
+        "articleType": "article-reference",
+        "href": "111111"
+      }
+    }];
+
+
+    const testManuscript = applyChangesToManuscript(mockManuscript, changesJSON);
+    expect(testManuscript.relatedArticles.length).toBe(2);
+
+    const updatedManuscript = applyChangesToManuscript(testManuscript, [{
+        "type": "delete-object",
+        "timestamp": 1614097785693,
+        "path": "relatedArticles",
+        "idField": "id",
+        "object": {
+          "_id": "ad319b14-c312-1111-a5a1-d07a548a6e7e",
+          "articleType": "article-commentary",
+          "href": "222"
+        }
     }]);
+
+    expect(updatedManuscript.relatedArticles.length).toBe(1);
+    expect(updatedManuscript.relatedArticles[0]).toEqual(changesJSON[1].object);
+  });
+
+  it('updates an entity on related article list', () => {
+    const changesJSON = [{
+      "type": "add-object",
+      "timestamp": 1614097785693,
+      "path": "relatedArticles",
+      "idField": "id",
+      "object": {
+        "_id": "ad319b14-c312-4627-a5a1-d07a548a6e7e",
+        "articleType": "article-reference",
+        "href": "111111"
+      }
+    }];
+
+    const testManuscript = applyChangesToManuscript(mockManuscript, changesJSON);
+
+    const updatedManuscript = applyChangesToManuscript(testManuscript, [{
+      "type": "update-object",
+      "timestamp": 1614250395860,
+      "path": "relatedArticles.0",
+      "differences": [{"kind": "E", "path": ["href"], "lhs": "111111", "rhs": "10.7554/eLife.4849811"}]
+    }]);
+    expect(updatedManuscript.relatedArticles.length).toBe(1);
+    expect(updatedManuscript.relatedArticles[0].id).toBe(changesJSON[0].object._id);
+    expect(updatedManuscript.relatedArticles[0].articleType).toBe(changesJSON[0].object.articleType);
+    expect(updatedManuscript.relatedArticles[0].href).toBe("10.7554/eLife.4849811");
+  });
+
+  it('applies a prosemirror change on the body', () => {
```

Comment:
This is good, shall we raise another issue for Title, Abstract, Impact Statment and Acknowledgment?

Reply:
Yeah I this would be good